### PR TITLE
Patricio problems 1 5

### PR DESCRIPTION
**Pull request by pvazquez93**
My solutions for the first 5 problems of Project Euler

### CODE REVIEW

File: problem5.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+"""
+Created on Sun Mar 18 10:34:22 2018
+@author: patricio
+
+is_divisible_by: given a number and a list of numbers, it returns true if the
+number is divisible by all the numbers in the list or false if not.
+"""
+import time
+
+def is_divisible_by(number, list_of_numbers):
```
Comment (shubags) [inline problem5.py]: This is kind of inefficient. Try using another method that utilizes prime numbers.

File: problem4.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+def get_all_prods(num_list1, num_list2):
+    cross_prod_list = list(((x,y) for x in num_list1 for y in num_list2))
+    mult_vec = set([multiply(lst) for lst in cross_prod_list])
+    return mult_vec
+
+nums = list(range(100,1000))
+mult_list = get_all_prods(nums,nums)
+
+palindrome_list = [num for num in mult_list if is_palindrome(num)]
```
Comment (shubags) [inline problem4.py]: Try to reduce the amount of unnecessary computations.

File: problem3.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        for i in range(2,number//2+1):
+            if number % i == 0:
+                default = False
+                break
+    return default
+
+def find_next_prime_num_after(num):
+    condition_meet = False
+    if num % 2 == 0:
+        k = num//2 #Since we're only checking numbers of the form 2k+1
```
Comment (shubags) [inline problem3.py]: Do you need to look this high?

File: problem2.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+"""
+
+def sum_if_mult_2(list_of_numbers):
+    sum_result = sum([i for i in list_of_numbers if i%2==0])
+    return sum_result
+
+def compute_fibonacci_seq_not_exceed(max_num):
+    fib_list = [1,2]
+    while (fib_list[len(fib_list)-1] <  max_num):
```
[the remark this answers is not in the document]
Comment (pvazquez93) [reply problem2.py]: Understood, I fixed it.

File: problem3.py (hunk 3 above)
Comment (pvazquez93) [reply problem3.py]: A different algorithm will be implemented, but no I don't need to look this high... lol

File: problem1.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+Description:
+This function calculates the sum of all multiples of 3 or 5 for a N number
+of numbers
+
+"""
+
+def sum_if_mult_3_5(num_nums):
+    sum_result = sum([i for i in range(num_nums) if i%3==0 or i%5==0])
+    return sum_result
+
```
[the remark this answers is not in the document]
Comment (pvazquez93) [reply problem1.py]: Understood, I fixed it

File: problem2.py [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    
+2) compute_fibonacci_seq_not_exceed: Computes the terms in the fibonbacci sequence 
+    whose value doesn't exceed the max_num paremeter
+
+"""
+
+def sum_if_mult_2(list_of_numbers):
+    sum_result = sum([i for i in list_of_numbers if i%2==0])
+    return sum_result
+
```
[the remark this answers is not in the document]
Comment (pvazquez93) [reply problem2.py]: Understood, I fixed it.

File: problem4.py (hunk 2 above)
Comment (pvazquez93) [reply problem4.py]: New implementation to reduce computations.

**Pull request merged**